modules/sd-nodes-cma-autoscaling-custom-install.adoc

```diff
@@ -126,7 +126,7 @@ spec:
         maxSize: "50"
   serviceAccount: {}
 ----
-<1> Specifies a single namespace where the custom autoscaler is to scale applications. Omit or set empty to scale applications in all namespaces. The default is empty.
+<1> Specifies a single namespace where the custom autoscaler is to scale applications. Omit or set empty to scale applications in all namespaces. This field should have one namespace or be empty. The default is empty.
```

Comment:
```suggestion
<1> Specifies a single namespace in which the custom autoscaler should scale applications. Leave it blank or leave it empty to scale applications in all namespaces. This field should have a namespace or be empty. The default value is empty.
```

Reply:
@agantony Thanks for your comments. It does read better.